modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|`ami-08b102eea4ca6bd02`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-04843cbe2741982bc`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0c22bbd807a0256d6`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0605f5746fd66f4fb`
+|`rhcos-45.82.202008010929-0`
+|`ami-0eb4e08ce1f20ca2e`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-053bbe72294a548d1`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0eef5030bf996ef45`
+|`rhcos-45.81.202005200134-0`
+|`ami-0e711f6fa6a408d08`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0ada217ae22ba5417`
+|`rhcos-47.83.202102090044-0`
+|`ami-094793700675782d2`
+|`rhcos-48.84.202107202156-0`
+|`ami-016bbb822d84af01d`
+|`rhcos-46.82.202008181646-0`
+|`ami-0f359d62ac28437e2`
+|`rhcos-49.84.202107191624-0`
+|`ami-0f1b225d48a937c73`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0f049ec45fb9e26bd`
+|`rhcos-48.84.202105062123-0`
+
+.19+|*ap-southeast-2*
+|`ami-0c744e459a8289065`
+|`rhcos-47.83.202102090044-0`
+|`ami-05db290d03ae42834`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0c313cb32b39ed6b7`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0503e1dacf597612b`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-065b26ab9484075b2`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-013b5b28f1b3cfc66`
+|`rhcos-46.82.202008181646-0`
+|`ami-0a12387dd4a2d1f10`
+|`rhcos-46.82.202008071940-0`
+|`ami-0fe73f4d11cbecc4d`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-081710aa569acd95a`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-06be0f08377760fc9`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-056be93d574739a4d`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-028c21b64920cb550`
+|`rhcos-49.84.202107191624-0`
+|`ami-015db805a21ad1367`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-01c8c1a3b43c3d128`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-004c7e92e3706ec63`
+|`rhcos-48.84.202107202156-0`
+|`ami-0e0a27dc586a9f242`
+|`rhcos-45.81.202005200134-0`
+|`ami-0c696c87e9199aa56`
+|`rhcos-45.82.202008010929-0`
+|`ami-0c50548f901b1c150`
+|`rhcos-48.84.202105062123-0`
+|`ami-0348c3bd1f92719ae`
+|`rhcos-414.92.202305090606-0-x86_64`
+
+.19+|*ap-northeast-1*
+|`ami-0f489be4932089df9`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0e5e4ab5bd66c69b3`
+|`rhcos-45.81.202005200134-0`
+|`ami-09cef6cbf25843f67`
+|`rhcos-47.83.202102090044-0`
+|`ami-0028d4d3fc213815f`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-00c6ece95bdb1f008`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-03d94dd5fe4f2c7e4`
+|`rhcos-48.84.202105062123-0`
+|`ami-049899b3091f4732c`
+|`rhcos-46.82.202008181646-0`
+|`ami-013fba3192d95f465`
+|`rhcos-46.82.202008071940-0`
+|`ami-0d8e30e0eba1db1ae`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-0d470a464b7276f8e`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0ec5f732a717b9d7e`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-0f48f7443cfdde283`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-079a33da05df563be`	
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?